### PR TITLE
Language sort optionlist at LED tab

### PR DESCRIPTION
**Pull request by HThuren**
Sort 2 optionlist using sortSelect -> Language sort

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

File: src/js/tabs/led_strip.js [hunk 1]
```diff
@@ -396,6 +396,10 @@ led_strip.initialize = function (callback, scrollPosition) {
             },
         });
 
+        const ledStripSelectFunction = $('#ledStripFunctionSelect');
+        // Sort the element, if need to group, do it by lexical sort, ie. by naming of (the translated) selection text
+        ledStripSelectFunction.sortSelect(i18n.getMessage("ledStripFunctionNoneOption"));
+
```
Comment (McGiverGim) [inline src/js/tabs/led_strip.js:402]: If we are only using the `ledStringSelectFunction` to sort it, better to do it in a single line.
```suggestion
        // Sort the element, if need to group, do it by lexical sort, ie. by naming of (the translated) selection text
        $('#ledStripFunctionSelect').sortSelect(i18n.getMessage("ledStripFunctionNoneOption"));

```
If we plan to use it for something more then is good to use the variable.

Comment (HThuren) [reply src/js/tabs/led_strip.js:402]: Good suggestion, i'll change to that.

**Review by haslinghuis — APPROVED** [review]

**Comment by HThuren** [comment]
Thank you

**Review by sugaarK — APPROVED** [review]

**Comment by HThuren** [comment]
Thank you

**Pull request merged**